packages/camera/camera_web/README.md

```diff
@@ -0,0 +1,5 @@
+# Camera Web Plugin
+
+A Flutter plugin for Web allowing access to the device cameras.
+
+*Note*: This plugin is under development.
```

Comment:
I'd add some instructions on how to add the plugin to your app, basically telling people that for now they need to depend *both* on `camera: ^x.y.z` and `camera_web: ^t.u.v`